lib/ecto/migrator.ex

```diff
@@ -153,6 +153,29 @@ defmodule Ecto.Migrator do
     end
   end
 
+  @doc """
+  Returns an array of tuples as the migration status of the given repo,
+  without actually running any migrations.
+
+  ## Options (same as the run command)
+
+    * `:all` - runs all available if `true`
+    * `:step` - runs the specific number of migrations
+    * `:to` - runs all until the supplied version is reached
+    * `:log` - the level to use for logging. Defaults to `:info`.
```

Comment:
Actually, it doesn't seem we use any of those options... but we probably should not anyway? :)